packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -147,7 +147,11 @@ const useTabs = () => {
       tab.name = user?.name || "my_account";
       tab.icon = undefined;
       tab.avatar = avatar?.avatar || WEBAPP_URL + "/" + session?.data?.user?.username + "/avatar.png";
-    } else if (tab.href === "/settings/security" && user?.identityProvider === IdentityProvider.GOOGLE) {
+    } else if (
+      tab.href === "/settings/security" &&
+      user?.identityProvider === IdentityProvider.GOOGLE &&
+      !user?.twoFactorEnabled
```

Comment:
We need to show 2FA option in settings for people with 2FA enabled even if they have a different provider than CAL, otherwise converted provider users couldn't disable 2FA